src/logic/safe/transactions/gas.ts

```diff
@@ -225,7 +218,38 @@ export const estimateSafeTxGas = async (
       dataGasEstimation,
     )
   } catch (error) {
-    console.error('Error calculating tx gas estimation', error)
+    console.info('Error calculating tx gas estimation', error.message)
```

Comment:
I think it should be a warning, not info.

Reply:
I discussed this with @fernandomg because we weren't sure it should be error or info, because is sort of expected error, what do you think about this @fernandomg ? I think warning it's ok as Nico suggested

Reply:
@nicosampler, why you think we should go with `warning` instead?

My reasons, why info is ok:
 - it's not an `error`, as we are handling it and we expect it to happen.
 - I don't think we should `warn` about every attempted gas estimation, is nothing wrong there, just a controlled scenario.
 - We are `informing` it as it's useful if anything wrong happens after this function call.

Reply:
makes sense